sled-agent/src/rack_setup/service.rs

```diff
@@ -866,6 +879,15 @@ impl ServiceInner {
         };
         let config = &plan.config;
 
+        // Initialize the trust quorum if there are peers configured.
+        if let Some(peers) = &config.trust_quorum_peers {
+            // The value doesn't really matter here. It just needs to be unique.
+            let rack_uuid = Uuid::new_v4();
```

Comment:
If this value doesn't matter, why is it exposed from the bootstore API? Why not just have this be an internal value within `init_rack`?

(There is another "rack_uuid" that we generate in RSS and send to Nexus, so this seems kinda confusing to have it re-defined)

Reply:
I exposed it specifically so that i could use the `rack_uuid` that gets generated in RSS and sent to Nexus, as that would help inform us which rack a sled came from easily if an initialized sled was place in another rack. I couldn't find where it was created, but I see now it's in the `SledPlan` which actually happens before this code. I will go ahead and use that UUID. Thank you for pointing this out!

Reply:
Fixed in [d953813](https://github.com/oxidecomputer/omicron/pull/3652/commits/d9538139aee296fd64b6db1813a2bd3573bff693)